src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h

```diff
@@ -101,20 +113,20 @@ typedef struct EnKaizoku {
     /* 0x578 */ Vec3s headRot;
     /* 0x580 */ Vec3f rightFootPos;
     /* 0x58C */ Vec3f leftFootPos;
-    /* 0x598 */ s32 unk_598;
-    /* 0x59C */ s16 unk_59C;
+    /* 0x598 */ s32 cutsceneTimer; // why is it double len?
```

Comment:
What do you mean by double len?

Reply:
Most timers are only s16, and even that is far larger than required, where most timers could be one unsigned byte without issues. Why would this one be double the bitsize?

Reply:
Felt like it was just preference of the developer. I would say the comment isn't really necessary though.